src/index.js

```diff
@@ -0,0 +1,233 @@
+devENV = false
+const baseURL = devENV ? 'http://localhost:5000' : 'http://christellenkera.com/weather-report-api'
+
+"use strict";
+// Common Selectors
+// const sky = document.getElementById("sky");
+const cityNameHeader = document.getElementById('headerCityName');
+const cityNameTextBox = document.getElementById('cityNameInput');
+const tempValue = document.getElementById("tempValue");
+const landscape = document.getElementById("landscape");
+const landscapeImg = document.getElementById("landscapeImg");
+
+// selects
+const skySelect = document.getElementById("skySelect");
+
+// buttons
+const resetCityName = document.getElementById("cityNameReset");
+const convertTempBtn = document.getElementById("convert-temp-btn");
+const increaseTempControl = document.getElementById("increaseTempControl");
+const decreaseTempControl = document.getElementById("decreaseTempControl");
+const currentTempButton = document.getElementById("currentTempButton");
+const citySearchButton = document.getElementById("citySearchButton");
+
+// default values
+const defaultCityNameInput = "Seattle";
+const defaultTempValue = 299.817; // 80 F
+const skies = {
+    chooseAForecast: "&nbsp;",
+    sunny: "☁️☁️☁️☀️☁️☁️☁️",
+    cloudy: "☁️☁️☁️☁️️☁️️☁️",
+    rainy: "🌧🌈🌦🌧🌧💧⛈",
+    snowy: "🌨❄️🌨🌨❄️❄️🌨",
+};
```

Comment:
👍 Nice, I like that you use a dictionary for this so that you can just access the value without needing brittle conditional logic with a bunch of if/else ifs